spec/calculator.spec.js

```diff
@@ -1,24 +1,31 @@
-'use strict';
+(function (factory) {
+  if (typeof require === 'function' && typeof exports === 'object' && typeof module === 'object') {
+    module.exports = factory(require('chai'), require('../Calculator'));
+  } else {
+    factory(chai, Calculator);
+  }
+} (function (chai, Calculator) {
```

Comment:
This is our factory function that wraps the tests. It takes in `chai` and `Calculator`